detectable/src/main/java/com/synopsys/integration/detectable/detectables/pip/parser/RequirementsFileDetectable.java

```diff
@@ -21,7 +21,7 @@
 import com.synopsys.integration.detectable.extraction.Extraction;
 import com.synopsys.integration.detectable.extraction.ExtractionEnvironment;
 
-@DetectableInfo(name = "PIP Requirements Parse", language = "Python", forge = "PyPi", accuracy = DetectableAccuracyType.LOW, requirementsMarkdown = "File: requirements.txt")
+@DetectableInfo(name = "PIP Requirements Parser", language = "Python", forge = "PyPi", accuracy = DetectableAccuracyType.LOW, requirementsMarkdown = "File: requirements.txt")
```

Comment:
"Parse" is used in other Detector names. It might make sense to keep "Parse" to stay consistent.

Reply:
That's unfortunate. Egregious even. ; )
I'll change that back.